plugins/analytics-module-ga/src/apis/implementations/AnalyticsApi/GoogleAnalytics.test.ts

```diff
@@ -508,4 +508,63 @@ describe('GoogleAnalytics', () => {
       expect(lastData.queueTime).toBeUndefined();
     });
   });
+
+  describe('api backward compatibility', () => {
+    it('continue working with legacy App category', () => {
+      const api = GoogleAnalytics.fromConfig(basicValidConfig);
+
+      expect(api.captureEvent).toBeDefined();
+
+      api.captureEvent({
+        action: 'navigate',
+        subject: '/',
+        context,
+      });
+
+      const [command, data] = ReactGA.testModeAPI.calls[1];
+      expect(command).toBe('send');
+      expect(data).toMatchObject({
+        hitType: 'pageview',
+        page: '/',
+      });
+    });
+
+    it('use lowercase app as the new default category', () => {
+      const api = GoogleAnalytics.fromConfig(basicValidConfig);
+
+      expect(api.captureEvent).toBeDefined();
+
+      api.captureEvent({
+        action: 'navigate',
+        subject: '/',
+        context: { ...context, extensionId: '', extension: '' },
+      });
+
+      const [command, data] = ReactGA.testModeAPI.calls[1];
```

Comment:
Same here (`ReactGA.testModeAPI.calls[0]` would contain the stuff in context that you could assert on)